addons/medical_treatment/functions/fnc_canStitch.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Katalam
+ * Prevents stitch actions from showing if the body is either fully sitched or has no open wounds.
+ *
+ * Arguments:
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ * 2: Body part <STRING>
+ * 3: Treatment class name <STRING>
+ *
+ * ReturnValue:
+ * Can Stitch <BOOL>
+ *
+ * Public: No
+ */
```

Comment:
Example is missing. Each header has to follow the same rules. For example: https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_canDig.sqf